packages/client/src/getExecutionContext.ts

```diff
@@ -25,6 +25,8 @@ export async function getConstExecutionContext(
     throw new Error("Provider should never be in fork mode and not in PR mode!");
   }
 
+  const isWithExitStatus: boolean = !!args.withExitStatus;
```

Comment:
Does it have to be part of `sharedExecutionContext`?  Idea behind execution context is to provide all important info to plugins. I can't think of a case when this would be valuable info for plugins as this is simply CLI argument.

Reply:
No, it doesn't have to be. I tried to use the abstraction that was already there without trying to come up with something else living next to it. Honestly, I was waiting for some comment like this one 😉